vms/platformvm/vm.go

```diff
@@ -219,7 +219,8 @@ func (vm *VM) Initialize(
 	vm.onShutdownCtx, vm.onShutdownCtxCancel = context.WithCancel(context.Background())
 	// TODO: Wait for this goroutine to exit during Shutdown once the platformvm
 	// has better control of the context lock.
-	go vm.Network.Gossip(vm.onShutdownCtx)
+	go vm.Network.PushGossip(vm.onShutdownCtx)
```

Comment:
Seems like these should be combined? Considered adding support for this in `gossip.Every` but got weird because frequency differed by `gossiper`.